include/libswiftnav/constants.h

```diff
@@ -70,6 +70,22 @@
 /** GPS C/A code chipping rate in Hz. */
 #define GPS_CA_CHIPPING_RATE 1.023e6
 
+/** GLO semi-major axis of Earth
+  * NOTE: there is define WGS84_A which is 6378137, differ than defined
+  * in GLO ICD, refer A.3.1.2. */
+#define GLO_A_E 6378136.0
+
+/** Second zonal harmonic of the geopotential */
+#define GLO_J02 1.0826257e-15
+
+/** Earth�s Gravitational Constant as defined in the GLO ICD in m^3 / s^2
+ * \note This is actually not identical to the usual WGS84 definition. */
+#define GLO_GM 3.9860044e14
```

Comment:
RTCM refers to ICD appendix A.3.1.2 so I used the value found there (398600.44).
Anyway the difference results in less than 0,0002 m change in computed track over 15 minutes.